src/plugins/stake-plugins/types.js

```diff
@@ -30,10 +30,7 @@ export type StakePolicy = {|
   stakeAssets: AssetId[],
 
   // The assets which can be earned
-  rewardAssets: AssetId[],
-
-  // Whether claim action is required to obtain reward
-  mustClaimRewards: boolean
```

Comment:
This was meant for FIO staking when we were to implement FIO staking using the staking plugin API

Reply:
So we should remove until if/when we do so again, yes?

Reply:
Yeah, that's fine. We can always look back in history as a reference to consider whether it's the right solution.